packages/sanity/src/structure/i18n/resources.ts

```diff
@@ -124,6 +124,9 @@ const structureLocaleStrings = defineLocalesResources('structure', {
   /** The action menu button tooltip */
   'buttons.action-menu-button.tooltip': 'Document actions',
 
+  /** The close button tooltip */
+  'buttons.close.tooltip': 'Close',
```

Comment:
I'd colocate this with the rest of the `document-inspector` resources, so maybe:
```suggestion
  /** Tooltip text shown for the close button of the document inspector */
  'document-inspector.close-button.tooltip': 'Close',
```

If so, this also needs to be sorted into a different area further down.

Reply:
Fair. I thought this was going to be more generic and needed in more places but those other instances were in `__workshop__` directories